easybuild/easyconfigs/o/OpenFOAM-Extend/OpenFOAM-Extend-3.2-gimkl-2.11.5.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'EB_OpenFOAM'
+
+name = 'OpenFOAM-Extend'
+version = '3.2'
+
+homepage = 'http://www.extend-project.de/'
+description = """OpenFOAM is a free, open source CFD software package. 
+OpenFOAM has an extensive range of features to solve anything from complex fluid flows
+involving chemical reactions, turbulence and heat transfer, 
+to solid dynamics and electromagnetics.""" 
+
+toolchain = {'name': 'gimkl', 'version': '2.11.5'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/Unofficial-Extend-Project-Mirror/foam-extend-foam-extend-%(version)s/archive']
+sources = ['v%(version)s.tar.gz']
+
+patches = [
+    'OpenFOAM-Extend-%(version)s-ParMGridGen.patch',
+    'OpenFOAM-Extend-3.1_build-qa.patch',
+    'OpenFOAM-Extend-3.1_comp-mpi.patch',
+    'OpenFOAM-Extend-3.1_skip-ThirdParty-OpenMPI.patch',
+]
+
+dependencies = [
+    ('hwloc', '1.11.1', '', ('GCC', '4.9.3')),
+    ('ParMETIS', '4.0.3'),
+    ('METIS', '5.1.0'),  # order matters, METIS need to be listed after ParMETIS to get $CPATH right
+    ('SCOTCH', '6.0.4'),
+    ('Mesquite', '2.3.0'),
+    ('ParMGridGen', '1.0'),
+    ('Python', '2.7.10'),
+    # Libccmio v2.6.1, zoltan v3.5
+]
+
+builddependencies = [
+    ('flex', '2.5.39'),
+    ('Bison', '3.0.4'),
+    ('M4', '1.4.17'),
+    ('CMake', '3.3.2'),
+]
+
+prebuildopts = "(wcleanAll || echo 'all is fine') && "  #  clean up everything first (and ignore any errors)
```

Comment:
yes please (both easyconfigs and easyblock, but do the easyconfigs on *after* this PR is merged)